tests/test_github.py

```diff
@@ -0,0 +1,70 @@
+from src.github import GitHubIssue
+import unittest
+from unittest.mock import patch
+
+
+def mocked_response(*args, **kwargs):
+    """Create class to mock response in _get_info method."""
+
+    class MockResponse:
+        def __init__(self, json_data):
+            self.json_data = json_data
+
+        def json(self):
+            return self.json_data
+
+    # Careful, args needs to be a tuple, and that always ends with a "," character in Python!!
+    if args == ('mock-url-REPO/issues/REAL-ISSUE',):
+
+        return MockResponse(
+            {'assignee': None,
+             'body': 'Issue Number: ABC-10',
+             'created_at': '2019-02-20T22:51:33Z',
+             'number': 100,
+             'state': 'open',
+             'title': 'Really an issue',
+             'updated_at': '2019-02-21T19:37:18Z',
+             'url': 'https://api.github.com/repos/ucsc-cgp/sync-agile-boards/issues/34',
+             'user': {'login': 'unito-bot'}}
+        )
+
+    elif args == ('mock-url-REPO/issues/NONEXISTENT-ISSUE',):
+        return MockResponse(
+            {'documentation_url': 'https://developer.github.com/v3/issues/#get-a-single-issue',
+             'message': 'Not Found'})
+
+    else:
+        raise RuntimeError(args, kwargs)
+
+
+def mocked_token():
```

Comment:
This is defined, but later you use the string anyway. I would do either one (and would prefer the latter, unless I'm missing something).

Reply:
fixed